blocksync/blocksync.go

```diff
@@ -291,9 +295,15 @@ func (bs *blockSyncer) ProcessBlock(ctx context.Context, peer string, blk *block
 		bs.lastTip = syncedHeight
```

Comment:
think we should also use `atomic.SwapUint64` to handle `bs.lastTip`

Reply:
`sync.RWMutex` is used above, so here no need